### PR TITLE
feat(slider): add docs for slider

### PR DESCRIPTION
**Pull request by CarolineGard**
This PR includes:
- Documentation for react
- Documentation for vue
- Documentation for usage
- Example

Not in this PR: 
- There are some questions about focusable behaviour. Will be fixed in another ticket
- Themable colors for on shadow for hover, active and focus. Current solution has the same color for all themes.

### CODE REVIEW

**Comment by AnnaRybkina** [comment]
Started with testing this out but I couldn't see any slider. I guess it should work right? 
image

File: docs/api-table.js [hunk 1]
```diff
@@ -792,18 +809,28 @@ export const vue = {
       ['as', 'string', 'form', 'The DOM element to emit for the wrapper'],
     ],
   },
+  Slider: {
+    required: [],
+    props: [
+      ['max', 'number', '100', 'The greatest value in the range of permitted values'],
+      ['min', 'number', '0', 'The lowest value in the range of permitted values'],
+      ['step', 'number', '', 'Specifies the value granularity'],
+      ['label', 'string', '', 'String value for aria-label the slider'],
```
Comment (AnnaRybkina) [inline docs/api-table.js]: Minor one, the description is a bit strange. Maybe we can skip `the slider`? Or add `of` before it? Or something else? Or is it just me? Feel free to skip if it doesn't make sense

```suggestion
      ['label', 'string', '', 'String value for aria-label'],
```
```suggestion
      ['label', 'string', '', 'String value for aria-label of the slider'],
```

File: docs/components/slider/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<script setup>
+  import React from './react.md';
+  import Vue from './vue.md';
+  import Usage from './usage.md';
+</script>
+
+# Slider
+
+A slider is an input where the user selects a value from within a given range. The precise value, however, is not necessarily considered important. It can have single or dual drag handles.
```
Comment (AnnaRybkina) [inline docs/components/slider/index.md]: Do we have examples with dual drag handles?

File: docs/api-table.js (hunk 1 above)
Comment (CarolineGard) [reply docs/api-table.js]: yeah, I agree! When I copied these from Fabric, I was thinking that we'll need to go through and improve the descriptions at some point. But I'll change those you mentioned now anyway:)

**Comment by CarolineGard** [comment]
> Started with testing this out but I couldn't see any slider. I guess it should work right? image

hm that's weird. Have you done `pnpm i`? or do you get any errors?
works for me
Screenshot 2023-05-12 at 09 45 19

File: docs/components/slider/index.md (hunk 2 above)
Comment (CarolineGard) [reply docs/components/slider/index.md]: copy paste miss, not yet supported. I will remove the sentence

File: docs/api-table.js (hunk 1 above)
Comment (AnnaRybkina) [reply docs/api-table.js]: Yea, we will for sure need to go through it all again 👍

**Review by AnnaRybkina — APPROVED** [review]

**Comment by AnnaRybkina** [comment]
Tested out again, slider showed up 👍 Great work!

**Pull request merged**